cdi/embedded/src/main/java/org/infinispan/cdi/InfinispanExtensionEmbedded.java

```diff
@@ -39,8 +39,8 @@
  * @author Pete Muir
  * @author Kevin Pollet <kevin.pollet@serli.com> (C) 2011 SERLI
  */
-public class InfinispanExtensionEmbedded {
-   private static final Log log = LogFactory.getLog(InfinispanExtensionEmbedded.class, Log.class);
+public class InfinispanExtensionEmbedded implements Extension {
+   private static final EmbeddedLog LOG = LogFactory.getLog(InfinispanExtensionEmbedded.class, EmbeddedLog.class);
```

Comment:
We never capitalize the log field, I guess references to `LOG` in the code tend to distract from the actual logic.